src/ProjectTemplates/Web.ProjectTemplates/content/BlazorServerWeb-CSharp/.template.config/template.json

```diff
@@ -9,10 +9,10 @@
   "generatorVersions": "[1.0.0.0-*)",
   "description": "A project template for creating a Blazor server app that runs server-side inside an ASP.NET Core app and handles user interactions over a SignalR connection. This template can be used for web apps with rich dynamic user interfaces (UIs).",
   "groupIdentity": "Microsoft.Web.Blazor.Server",
-  "precedence": "7000",
-  "identity": "Microsoft.Web.Blazor.Server.CSharp.5.0",
+  "precedence": "8000",
+  "identity": "Microsoft.Web.Blazor.Server.CSharp.6.0",
   "shortName": "blazorserver",
-  "thirdPartyNotices": "https://aka.ms/aspnetcore/5.0-third-party-notices",
+  "thirdPartyNotices": "https://aka.ms/aspnetcore/6.0-third-party-notices",
```

Comment:
This currently resolves to microsoft.com, whereas the 5.0 3rd party notices goes somewhere real.  This value us unrelated to template creation in Visual Studio. If we are going to update the aka.ms link later, this is probably fine.

Reply:
@phenning I think so, but these need to be updated for 6.0 in the templates no matter what.

Reply:
I've already created the mapping for the `to-be-created` branch.